File: BoletosAPI/Models/Usuarios.cs
namespace BoletosAPI.Models
{
    public class Usuarios
    {
        public int UserId { get; set; }
        public string UserNombre { get; set; } = String.Empty;
        public string UserEmail { get; set; } = String.Empty;
        public string UserClave { get; set; } = String.Empty;
    }
}

File: BoletosDesktop/Models/Usuarios.vb
Imports System.Text.Json.Serialization

Public Class Usuarios
    <JsonPropertyName("userId")>
    Property UserId As Integer
    <JsonPropertyName("userNombre")>
    Property UserNombre As String
    <JsonPropertyName("userEmail")>
    Property UserEmail As String
    <JsonPropertyName("userClave")>
    Property UserClave As String
End Class
